Gemfile.lock

```diff
@@ -0,0 +1,93 @@
+PATH
```

Comment:
Do we need to commit this file? Doesn't seem necessary for gems?

Reply:
Gemfile.locks should not be committed in Gems. It should be added to your gitignore.

Explanation...old but still true: http://yehudakatz.com/2010/12/16/clarifying-the-roles-of-the-gemspec-and-gemfile/